CHANGES.rst

```diff
@@ -222,6 +222,11 @@ Bug Fixes
 
 - ``astropy.io.fits``
 
+  - Handle unicode FITS BinTable column names on Python 2 [#5204, #4805]
```

Comment:
I believe  this line is a remnant from somewhere (that PR was merged into 1.2.2) and should be removed.